yjit_codegen.c

```diff
@@ -3732,6 +3733,7 @@ gen_send_iseq(jitstate_t *jit, ctx_t *ctx, const struct rb_callinfo *ci, const r
         mov(cb, mem_opnd(64, REG0, sizeof(VALUE) * (i - num_locals - 3)), imm_opnd(Qnil));
     }
 
+    ADD_COMMENT(cb, "push EP");
     // Put compile time cme into REG1. It's assumed to be valid because we are notified when
     // any cme we depend on become outdated. See rb_yjit_method_lookup_change().
     jit_mov_gc_ptr(jit, cb, REG1, (VALUE)cme);
```

Comment:
You added an ASM comment here that says "push EP" but the next line is storing the CME?

Reply:
Right. I guess this isn't pushing EP but the ENV EP points to (for which cme is the first element). I can update this